components/Newsfeed/NotificationProps.tsx

```diff
@@ -1,22 +1,41 @@
 import { Timestamp } from "firebase/firestore"
 
-export type NotificationProps = {
+export type NotificationProps = BillHistoryNotification | TestimonyNotification
+
+type BillHistoryNotification = {
+  type: "bill"
+  bodyText: string
+  court: string
+  delivered: boolean
+  header: string // Bill Title
```

Comment:
If we're doing more specific notification types now, could we just pass along `billTitle` and trust the frontend to display what it needs to do with it?